metricbeat/module/elasticsearch/node_stats/data.go

```diff
@@ -123,6 +199,14 @@ func eventsMapping(r mb.ReporterV2, content []byte) error {
 			r.Error(err)
 		}
 
+		// ingest.pipelines.*
+		value, ok := node["ingest"].(map[string]interface{})["pipelines"]
```

Comment:
@ruflin Is this multi-level indexing into the map + type assertion, all in one step, safe to do or should I break this up into 3 separate steps (index into "ingest", then the type assertion, then index into "pipelines") with error checking after each step?